### DISCUSSION TITLE
Future directions?

### DISCUSSION THREAD

**Discussion by conorhearn1** [haotian-liu/LLaVA#43] [Ideas]
Really fascinating work, great job! I would be fascinated to know what future directions you're planning to explore. Are you thinking of utilizing SAM or other vision encoders in future models? Would also be interesting to see how performance is affected by the model size, such as how larger LLaMA models perform with this scheme... The dataset is phenomenal work, thank you!

**Comment by ChunyuanLI** [comment]
There are many exciting directions enabled after we see LLaVA achieves the amazing results with little data and training. Please see the very brief discussions on the future opportunities we wrote in the paper:

![image](https://user-images.githubusercontent.com/8978644/233802936-4ec54d8c-9730-4f0d-8244-3e16f980a9a3.png)
